changelogs/fragments/collections.yml

```diff
@@ -0,0 +1,4 @@
+major_changes:
+- Experimental support for Ansible Collections and content namespacing - Ansible content can now be packaged in a
```

Comment:
Use mdash
```suggestion
- Experimental support for Ansible Collections and content namespacing — Ansible content can now be packaged in a
```

Reply:
we're not doing that anywhere else in changelogs that I'm aware of, so keeping is consistent for now...